.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

Let's ignore pushing all possible log files
```suggestion
logs/*
```